commit 3c918c14a2fdf630ee76a24ef5380cc1e3eb31e3
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Oct 29 15:50:50 2018 +0000

    ARM: bcm2835_defconfig: Enable bcm2835-camera
    
    Enables the V4L2 camera driver as a module.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/arch/arm/configs/bcm2835_defconfig b/arch/arm/configs/bcm2835_defconfig
index e9bc88937b1e..5a61367bb47c 100644
--- a/arch/arm/configs/bcm2835_defconfig
+++ b/arch/arm/configs/bcm2835_defconfig
@@ -91,6 +91,8 @@ CONFIG_THERMAL=y
 CONFIG_BCM2835_THERMAL=y
 CONFIG_WATCHDOG=y
 CONFIG_BCM2835_WDT=y
+CONFIG_MEDIA_SUPPORT=m
+CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_DRM=y
 CONFIG_DRM_VC4=y
 CONFIG_FB_SIMPLE=y
@@ -128,7 +130,8 @@ CONFIG_LEDS_TRIGGER_CAMERA=y
 CONFIG_DMADEVICES=y
 CONFIG_DMA_BCM2835=y
 CONFIG_STAGING=y
-CONFIG_BCM2835_VCHIQ=m
+CONFIG_SND_BCM2835=m
+CONFIG_VIDEO_BCM2835=m
 CONFIG_MAILBOX=y
 CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1259_3c918c14a2fdf630ee76a24ef5380cc1e3eb31e3.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1259_3c918c14a2fdf630ee76a24ef5380cc1e3eb31e3.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
